client/blocks/inline-help/contextual-help.js

```diff
@@ -766,329 +869,371 @@ const videosForSection = {
+			title: translate( 'Add a Contact Form to Your Website' ),
+			description: translate( 'Find out how to add a contact form to your WordPress.com site.' ),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/ssfHW5lwFZg',
-			title: 'Embed a YouTube Video in Your Website',
-			description:
+			title: translate( 'Embed a YouTube Video in Your Website' ),
+			description: translate(
 				'Find out how to embed a YouTube video in your content (including posts, pages, and even comments) ' +
-				'on your WordPress.com or Jetpack-enabled WordPress website or blog.',
+					'on your WordPress.com or Jetpack-enabled WordPress website or blog.'
+			),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/_tpcHN6ZtKM',
-			title: 'Schedule a Post',
-			description: 'Find out how to schedule a post on your WordPress.com website or blog.',
+			title: translate( 'Schedule a Post' ),
+			description: translate(
+				'Find out how to schedule a post on your WordPress.com website or blog.'
+			),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/V8UToJoSf4Q',
-			title: 'Add a Simple Payment Button',
-			description: 'Find out how to add a payment button to your WordPress.com website.',
+			title: translate( 'Add a Simple Payment Button' ),
+			description: translate(
+				'Find out how to add a payment button to your WordPress.com website.'
+			),
 		},
 	],
 	account: [
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/aO-6yu3_xWQ',
-			title: 'Change Your Password',
-			description: 'Find out how to change your account password on WordPress.com.',
+			title: translate( 'Change Your Password' ),
+			description: translate( 'Find out how to change your account password on WordPress.com.' ),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/qhsjkqFdDZo',
-			title: 'Change Your WordPress.com Username',
-			description: 'Find out how to change your username on WordPress.com.',
+			title: translate( 'Change Your WordPress.com Username' ),
+			description: translate( 'Find out how to change your username on WordPress.com.' ),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/Tyxu_xT6q1k',
-			title: 'Change Your WordPress.com Display Name',
-			description: 'Find out how to change your display name on WordPress.com.',
+			title: translate( 'Change Your WordPress.com Display Name' ),
+			description: translate( 'Find out how to change your display name on WordPress.com.' ),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/07Nf8FkjO4o',
-			title: 'Change Your Account Email Address',
-			description: 'Find out how to change your account email address on WordPress.com.',
+			title: translate( 'Change Your Account Email Address' ),
+			description: translate( 'Find out how to change your account email address on WordPress.com.' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Find out how to change your account email address WordPress.com.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).